### PR TITLE
Implement multi controller

### PR DESCRIPTION
**Pull request by tobiaslabu**
This PR aims to implement support for multiple controller drivers. See #11.

### CODE REVIEW

File: src/config/controller.rs [hunk 1]
```diff
@@ -24,9 +25,46 @@ use super::midi::{self, ToMidiEvent};
...
+            config.next.to_midi_event()?,
+            config.stop.to_midi_event()?,
+            config.all_songs.to_midi_event()?,
+            config.playlist.to_midi_event()?,
+        ))),
+        None => Err("No MIDI device found for MIDI controller.".into()),
+    }
+}
+
+fn driver_from_midi_config2(
```
Comment (mdwn) [inline src/config/controller.rs]: Could we coalesce this and the other driver from midi config function into one somehow? If not, could we rename this?

**Review by mdwn — COMMENTED** [review]
This is great! I've got one piece of feedback, but otherwise I think this is perfect!

File: src/config/controller.rs (hunk 1 above)
Comment (tobiaslabu) [reply src/config/controller.rs]: Thanks for the feedback! I intended this to be only one function but I could not get the compiler to agree yet 😆. I'll give it another shot, I don't like the current solution either. 

Edit: Found a solution, see 41591baff7e7b22615792d13dd070a586d9a6e4e

**Review by mdwn — APPROVED** [review]
Thank you!

**Pull request merged**